content/docs/release-notes/archive/22-05-0/_index.md

```diff
@@ -0,0 +1,96 @@
+  * O3DE's AssImp integration now matches other 3rd party libraries for O3DE. ([3p-package-source/75](https://github.com/o3de/3p-package-source/pull/75))
+  * Full scan time for Automated Testing is down from ~2.5min to ~40 seconds. ([6619](https://github.com/o3de/o3de/pull/6619))
+  * Job Dependencies on Products: Allows Asset Builders to provide a list of products they specifically depend on to better schedule and control create jobs flow in the Asset Processor. ([RFC](https://github.com/o3de/sig-core/issues/28))
+* **Character**
+  * Atom Render Viewport: The animation editor now uses Atom. This improves performance significantly, the rendering in the animation editor is now comparable to what one would expect in-game, and the animation editor viewport is now consistent with other O3DE viewports. The legacy OpenGL render viewport for the animation editor is now deprecated.
+* **Editor**
+  * Generic DOM: Core framework update to support the Document Property Editor.
+* **Entities & Prefabs**
+  * Improved Script Canvas support for Spawning: Previously, it was only possible to spawn entities using Script Canvas with `SpawnAllEntities`. This feature expands Spawnable API functionality for Script Canvas. It provides more fine-grained control over spawning workflow and spawnable lifecycle as well as better support for handling multiple prefabs.
+  * Slice deprecation in automated testing: The AutomatedTesting project has been converted to use prefabs for all tests (previously there were many that used the previous system, Slices). ([7006](https://github.com/o3de/o3de/issues/7006))
+* **Physics**
+  * Remove legacy timing/tick system: The legacy tick/timer system has been completely removed and new timer for O3DE has been added. All systems have been updated to use the new timer. This serves as the foundation for future tick system updates and improvements. Previous smoothing logic was also removed.
+  * Upgrade Physics Automated Tests:All physics automated tests have been updated to use Prefabs instead of Slices (including PhysX, Cloth and Blast). All tests are using the new test framework (in most cases). Several are also now running in parallel.
+* **Installer**
+  * Installer validation tests: These tests verify that an installer build is valid. The automated tests verify that key binaries exist, that o3de.exe registers engine, create project succeeds, compile project succeeds, asset processor batch succeeds, editor runs, game launcher runs, uninstall succeeds and unregisters engine. Currently run as nightly for O3DE, though the tests are designed so anyone can plug these tests into their quality verification process. ([7834](https://github.com/o3de/o3de/pull/7834))
+* **Script Canvas**
+  * Script Canvas now has a "mini map" feature that helps visualize and navigate graphs. ([6263](https://github.com/o3de/o3de/pull/6263))
+* **Viewport**
```

Comment:
Adding late, approved addition to the release notes (default component template).
Sig-release PR: https://github.com/o3de/sig-release/pull/56
```suggestion
* **Templates**
  * Default component template: A new template is available to help you create components. Use the `o3de` CLI script and the `create-from-template` command to start a runtime component from the **DefaultComponent** template. For detailed instructions refer to [Creating a Component in O3DE](/docs/user-guide/programming/components/create-component/). ([8462](https://github.com/o3de/o3de/pull/8462))
* **Viewport**
```